data/saas/config/mailchimp_transactional_config.yml

```diff
@@ -0,0 +1,46 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mailchimp Transactional SaaS Config
+  type: mailchimp_transactional
+  description: A sample schema representing the Mailchimp Transactional (Mandrill) connector for Fides
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+      default_value: mandrillapp.com/api/1.0
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: api_key
+      configuration:
+        body: |
+          {
+            "key": "<api_key>"
+          }
+
+  test_request:
+    method: GET
+    path: /users/ping
+
+
+  consent_requests:
+    opt_in:
+      method: POST
+      path: /allowlists/add
+      body: |
+        {
+          "email": "<email>"
+        }
+
+    opt_out:
+      method: POST
+      path: /allowlists/delete
+      body: |
+        {
+          "email": "<email>"
+        }
```

Comment:
Format taken from here: https://github.com/ethyca/fides/issues/2144#issuecomment-1372860588, except I'm making an opt_in/opt_out distinction.  This is just a tentative format to have something to demonstrate, I need to do a more broad survey in https://github.com/ethyca/fides/issues/2144.

Right now (design decision by Neville) the Privacy Center will restrict so only one consent preference can be considered executable at any given time, so by the time this gets to consent request execution, there will be at most one consent preference, with opt-in or opt-out that we will fire accordingly.

There is currently no logic linking which requests are made for which types of data uses but this is part of a broader discussion about where this should happen between multiple teams.

Reply:
I like the opt_in/opt_out distinction, it makes the purpose of these requests very clear